src/Controller/ContactFormController.php

```diff
@@ -73,9 +79,14 @@ public function createContactMessage(Request $request): Response
                 $contact->setSendTime(new \DateTime());
                 $this->entityManager->persist($contact);
                 $this->entityManager->flush();
-                $SM = $this->adminUserRepository->find(1);
-                assert($SM instanceof AdminUser);
-                $this->mailer->send('contact_shop_manager_mail', [$SM->getEmail()], ['contact' => $contact]);
+
+                $channel = $this->channelContext->getChannel();
+                Assert::isInstanceOf($channel, ChannelInterface::class);
```

Comment:
use it as always `assert($channel instanceof ChannelInterface);`

Reply:
DONE